File: lua/refactoring/tests/refactor/106/go/struct/extract.expected.go
package main

import "fmt"

type foobar struct {
    poggers int
}

func (f *foobar) foo(a int, test int, test_other INSERT_PARAM_TYPE) {
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(f.poggers, idx, a)
    }
}

func (f *foobar) simple_function(a int) {
    var test int = 1
    test_other := 1
    f.foo(a, test, test_other)
}

File: lua/refactoring/tests/refactor/106/java/prompt_func_param/extract.expected.java
class Extract {

public static void foo(int a) {
    int test_other = 1;

    for (int i = 0; i < test_other; i++) {
      System.out.printf("%d %d", a, test_other);
    }
}


  public static int simpleFunction(int a) {
    int test = 1;
    foo(a);

    return test;
  }
}
